conductr_cli/license.py

```diff
@@ -78,10 +82,16 @@ def format_multiple(key, title_text):
 def get_license(args):
```

Comment:
I think usage would be clearer if we returned a 2 value tuple - (forgive the Scala type declaration :p)
   `(Boolean, Option[Json])` - and then we wouldn't need a special constant either.

So not supported returns: `False, None`
Supported, but no data: `True, None`
Supported with data: `True, Json`